cli/cmd/root.go

```diff
@@ -154,18 +153,30 @@ func NewCmdRoot() *cobra.Command {
 			panic(err)
 		}
 	}
-	initViper()
+	initViper(cmd)
 	cmd.SetHelpCommand(&cobra.Command{Hidden: true})
 	cmd.AddCommand(NewCmdSync(), newCmdDoc())
 	cmd.CompletionOptions.HiddenDefaultCmd = true
 	cmd.DisableAutoGenTag = true
 	return cmd
 }
 
-func initViper() {
-	viper.AutomaticEnv()
```

Comment:
Dropped this in favor of manual configuration. I think configuring it manually makes it much easier to understand which Env variable to use (and also allows us to customize the name)

Reply:
Great. See I did the same here - https://github.com/cloudquery/cloudquery/pull/2619

Reply:
EDIT: I it's better to drop viper completely it's a horrible library. Also, I don't think environment variable is needed for `--cq-dir` if someone will ask for it we can always add it.

Currently only added for telemetry as it's common to put it in the `.bashrc`